jbmc/src/java_bytecode/select_pointer_type.h

```diff
@@ -36,6 +36,16 @@ class select_pointer_typet
 
 public:
   virtual ~select_pointer_typet() = default;
+
+  /// Select what type should be used for a given pointer type. For the base
+  /// class we just use the supplied type. Derived classes can override this
+  /// behaviour to provide more sophisticated type selection. Generic
+  /// parameters are replaced with their concrete type.
+  /// \param pointer_type: The pointer type replace
+  /// \param generic_parameter_specialization_map map of types for all generic
+  /// parameters in the current scope
```

Comment:
:pick: It would be more readable if this had an indent. Same for the `\param`s in `specialize_generics`.
:frog:

Reply:
Done